Baconit/Strings/es-es/Resources.resw

```diff
@@ -0,0 +1,889 @@
+  <data name="CBIpweek.Content" xml:space="preserve">
+    <value>La semana pasada</value>
+  </data>
+  <data name="CBIpyear.Content" xml:space="preserve">
+    <value>El año pasado</value>
+  </data>
+  <data name="CBIrising.Content" xml:space="preserve">
+    <value>Tendencia</value>
+  </data>
+  <data name="CBItop.Content" xml:space="preserve">
+    <value>Lo mejor</value>
+  </data>
+  <data name="CCommentsCBB.Content" xml:space="preserve">
+    <value>Comentarios</value>
+  </data>
+  <data name="CControCB.Content" xml:space="preserve">
+    <value>Controvérsico</value>
+  </data>
+  <data name="CheckUnder.Text" xml:space="preserve">
+    <value>Código fuente</value>
+  </data>
+  <data name="CNewCB.Content" xml:space="preserve">
+    <value>Nuevo</value>
+  </data>
+  <data name="COldCB.Content" xml:space="preserve">
+    <value>Viejo</value>
+  </data>
+  <data name="CPastDayB.Content" xml:space="preserve">
+    <value>Ayer</value>
+  </data>
+  <data name="CPastHourB.Content" xml:space="preserve">
+    <value>La última hora</value>
+  </data>
+  <data name="CPastMonthB.Content" xml:space="preserve">
+    <value>El mes pasado</value>
+  </data>
+  <data name="CPastWeekB.Content" xml:space="preserve">
+    <value>La semana pasada</value>
+  </data>
+  <data name="CPastYearB.Content" xml:space="preserve">
+    <value>El año pasado</value>
+  </data>
+  <data name="CQACB.Content" xml:space="preserve">
+    <value>Q&amp;A</value>
+  </data>
+  <data name="CreateAccountCT.Content" xml:space="preserve">
+    <value>Crea una cuenta</value>
+  </data>
+  <data name="CSettings.Text" xml:space="preserve">
+    <value>Configuración de los comentarios/value>
+  </data>
+  <data name="CTopCB.Content" xml:space="preserve">
+    <value>Lo mejor</value>
+  </data>
+  <data name="DevSettings.Text" xml:space="preserve">
+    <value>Configuración de desarrollador</value>
+  </data>
+  <data name="DisableCombo.Content" xml:space="preserve">
+    <value>Deshabilitado por subreddit; Reseta cuando la app este cerrada.</value>
+  </data>
+  <data name="DisablePostLoad.Text" xml:space="preserve">
+    <value>No se carga el contenido hasta que se pulse la pantalla.</value>
+  </data>
+  <data name="EverythingCB.Content" xml:space="preserve">
+    <value>Todo</value>
+  </data>
+  <data name="FatalException.Text" xml:space="preserve">
+    <value>No dejes que las excepciones fatales maten la app, informa de las excepciones en lugar de romper.(Esto dejará la app en un estado inestable)</value>
+  </data>
+  <data name="FlipViewS.Text" xml:space="preserve">
+    <value>Configuración de la vista</value>
+  </data>
+  <data name="fullpost.Text" xml:space="preserve">
+    <value>Muestra todos los títulos del post.</value>
+  </data>
+  <data name="HeaderAboutTB.Text" xml:space="preserve">
+    <value>Sobre</value>
+  </data>
+  <data name="HowMessagesTB.Text" xml:space="preserve">
+    <value>Como los mensajes aparecerán en el centro de notificaciones.</value>
```

Comment:
This is ok.
You could also say "Como los mensajes aparecen...". Any of both are ok.